doc/patchutils.xml

```diff
@@ -238,6 +239,14 @@
 	    </listitem>
 	  </varlistentry>
 
+	  <varlistentry>
+	    <term><option>--color=<replaceable>WHEN</replaceable></option></term>
+	    <listitem>
+	      <para>colorize the output; <replaceable>WHEN</replaceable>
+		can be 'never', 'always', or 'auto' (the default)</para>
```

Comment:
Does passing through `'auto'` make sense here? I feel like it will function the same as `'never'` because there is no terminal.